server/services/healthcheck.js

```diff
@@ -13,8 +13,13 @@ const service = (name, url) => {
       .catch(err => ({ name, status: 'ERROR', message: err }))
 }
 
-module.exports = function healthcheckFactory(authUrl, elite2Url) {
-  const checks = [db, service('auth', `${authUrl}/ping`), service('elite2', `${elite2Url}/ping`)]
+module.exports = function healthcheckFactory(authUrl, elite2Url, tokenVerificationUrl) {
+  const checks = [
+    db,
+    service('auth', `${authUrl}/ping`),
+    service('elite2', `${elite2Url}/ping`),
```

Comment:
guess these should probably call /heath/ping too

Reply:
Done